DATA/production/qc-workflow.sh

```diff
@@ -153,15 +154,34 @@ elif [[ -z $QC_JSON_FROM_OUTSIDE ]]; then
     fi
   done
 
-  LIST_OF_GLOQC=
-  has_detectors_reco ITS TPC && has_detector_matching ITSTPC && add_comma_separated LIST_OF_GLOQC ITSTPC
-  has_detectors_reco ITS && has_detector_matching PRIMVTX && add_comma_separated LIST_OF_GLOQC PRIMVTX
-  for i in $(echo $LIST_OF_GLOQC | sed "s/,/ /g"); do
-    GLO_JSON_FILE="QC_JSON_$i"
-    if has_detector_matching $i && has_matching_qc $i && [ ! -z "${!GLO_JSON_FILE}" ]; then
-      add_QC_JSON $i ${!GLO_JSON_FILE}
+  ADD_ITSTPCQC=1
+  ADD_PRIMVTX=1
+  if [[ $SYNCMODE == 1 ]]; then
+    if ! has_detectors_reco ITS TPC || ! has_detector_matching ITSTPC; then
+      QC_CONFIG+="--override-values \"qc.tasks.MTCITSTPC.active=false\""
+      ADD_ITSTPCQC=0
     fi
-  done
+    if ! has_detectors_reco ITS || ! has_detector_matching PRIMVTX; then
+      QC_CONFIG+="--override-values \"qc.tasks.Vertexing.active=false\""
```

Comment:
I don't think we accept multiple `--override-values` in one command. You probably have to assemble the argument's value in one step and add it at the end.

Reply:
ok, but that was the reasoning behind having itstpc, and another one for every other detector matching. So I am not sure if this will work out with our current framework. Otherwise I am agnostic wrt. the JSON setup, but we have to make sure to keep a flexible structure that will work with each combination of detectors.